facebook.py

```diff
@@ -91,6 +89,9 @@ class FacebookPage(models.Source):
   username = ndb.StringProperty()
   # inferred from syndication URLs if username isn't available
   inferred_username = ndb.StringProperty()
+  # inferred application-specific user IDs (from other applications)
+  inferred_app_scoped_user_ids = ndb.StringProperty(repeated=True)
```

Comment:
ah, that's a really good point, like if bridgy has an app-scoped ID and the POSSE application has a global ID.